include/rcutils/stdatomic_helper/win32/stdatomic.h

```diff
@@ -186,6 +186,12 @@ typedef _Atomic (ptrdiff_t) atomic_ptrdiff_t;
 typedef _Atomic (intmax_t) atomic_intmax_t;
 typedef _Atomic (uintmax_t) atomic_uintmax_t;
 
+#ifdef ROS_PACKAGE_NAME
+  #define RCUTILS_COPY_NAME ROS_PACKAGE_NAME
```

Comment:
Why "copy" in the name?

Also I would lean toward using a leading underscore so that this local define doesn't conflict with anything else.

Maybe `_RCUTILS_PACKAGE_NAME`?

Reply:
> Why "copy" in the name?

:man_shrugging: 

> Maybe _RCUTILS_PACKAGE_NAME?

Done d5f48b4